### DISCUSSION TITLE
New problem with video downloadhelper

### DISCUSSION THREAD

**Discussion by dverdier** [aclap-dev/video-downloadhelper#231] [Help and Q&A]
### Your question

I am not able anymore to download videos since a couple of weeks. 
I got MacOs Sonoma Version 14.3.1 (23D60)
Please have a look on the attached files (those files are inoperative....
<img width="512" alt="Capture d’écran 2024-02-28 à 19 58 47" src="https://github.com/aclap-dev/video-downloadhelper/assets/161640535/fc77f7d5-2d3d-4ae7-ac83-6d727084b229">

**Comment by paulrouget** [comment]
I can't help you without details.

I need:
- URL of the page you're downloading from
- details about your setup. Please read [this](https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F) and do exactly what it says.
- what is the error exactly

**Reply by dverdier** [reply]
version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: fr
license: accepted
key (only 16 first characters): ca39187217cd4f26
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac13",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/davidverdier",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.1.1-tessus"
}

**Reply by dverdier** [reply]
the URL I try to download : https://www.cinematheque.fr/henri/film/154795-opera-glasses-mila-zhluktenko-2019/ 

but any URL....

**Reply by dverdier** [reply]
the problem is that Video Downloadhelper gives me 3 links (see insert files) 
<img width="513" alt="Capture d’écran 2024-03-03 à 00 09 36" src="https://github.com/aclap-dev/video-downloadhelper/assets/161640535/429bf609-bb97-4189-9903-a16129d2f1e4">
but I cannot download

**Reply by GrampaWildWilly** [reply]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3235, Firefox 123.0 64-bit, licensed VDH 8.2.2.0a1 beta, CoApp 2.0.19.  I do not work for VDH.  I am just another user, like you.

I visited your link & got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/572d15bd-7bac-4766-a03f-b012b83964c7)

If these images are too small to read, just click on the image.  That opens the image in its own browser window.  Then you can use the browser magnify tool (Ctrl-+) to make things legible.

While it was downloading, this is what things looked like:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e760c45b-4f27-4d52-93b4-5c72c14c2ecf)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/a1548160-a749-4cb7-b9df-4055d07ea40f)

Message for @paulrouget:  Look at the download speed VDH is reporting & compare it to the numbers in the Resource Monitor.  You'll notice in the "TCP Connections" pane at the bottom that there are multiple instances of ffmpeg.  You appear to be reporting one of those numbers.  But the true download speed is the one showing in the "Processes with Network Activity" pane at the top.

Once the download completed, which it did successfully, this was my result:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/dfa2c3d4-78b3-4db8-9ad7-115e2f63ed0d)

Here it is playing in VLC:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/172a5d80-44ce-4919-ae0a-163be58fc0e4)

You can see it is playing fine.  You'll have to take my word for it that the audio was also fine.  I did not sit & watch this, just sampled it at intervals.  It was all there.

I think the conclusion is you should be using the same software.  It looks like you are for everything except the browser extension itself.  And the operating system, which should not matter.  You need to upgrade to the latest release.  You can find that by clicking the Wiki link at the top of this page.  That will take you to the Wiki, where there is a Table of Contents down the right side of the page.  Go to the chapter that discusses beta versions & get the latest VDH beta.  You should then get the same results I got.
